vehicle/src/Vehicle/Backend/LossFunction/Compile.hs

```diff
@@ -217,6 +213,197 @@ compileQuant :: V.Quantifier -> Quantifier
 compileQuant V.Forall = All
 compileQuant V.Exists = Any
 
+data Translation = Translation
```

Comment:
Tried moving it, but then I have to import things such as LExpr from LossFunction.Compile and it errors out because of cyclic module dependency between these two. Is there any solution to that?

Reply:
I think the answer is that we should move `LExpr` in there as well?

Reply:
Done in one of the latest commits 👍